modules/Story/Story.tsx

```diff
@@ -12,19 +12,20 @@ const Story: FunctionComponent<Props> = ({ story }) => {
     }
 
     const {
-        title, subtitle, content, format_version, htmlContent,
+        title, subtitle, content, format_version,
     } = story;
 
     return (
         <article>
             <h2>{title}</h2>
             <h3>{subtitle}</h3>
             {format_version === 1 && (
```

Comment:
Probably it's a good moment to get rid of magic numbers and creating an enum or at least verbose variables for this. Since it's very unclear what is `1` and what is `3`

Reply:
Done. The sdk already comes with the enum.